next/components/forms/segments/AccountSectionHeader/AccountSectionHeader.tsx

```diff
@@ -0,0 +1,21 @@
+import { SectionItemBase } from 'components/forms/types/AccountTypes'
+import { useRouter } from 'next/router'
+import React from 'react'
+
+type AccountSectionHeaderBase = {
+  sectionsList: SectionItemBase[]
+}
+
+const AccountSectionHeader = (props: AccountSectionHeaderBase) => {
```

Comment:
I'm confused as to why we need both this and the list in AccouintNavBar

one of them is the header menu - what is the other one for ?

Reply:
AccountSectionHeader is part of each subpage